accounting/entries.go

```diff
@@ -241,3 +241,52 @@ func invoiceEntry(invoice *lnrpc.Invoice, circularReceipt bool,
 		hash, preimage, note, false, convert,
 	)
 }
+
+// forwardTxid provides a best effort txid using incoming and outgoing channel
+// ID paired with timestamp in an effort to make txid unique per htlc forwarded.
+// This is not used as a reference because we could theoretically have duplicate
+// timestamps.
+func forwardTxid(chanIn, chanOut, unixTx uint64) string {
+	return fmt.Sprintf("%v:%v:%v", chanIn, chanOut, unixTx)
+}
+
+// forwardNote creates a note that indicates the amuonts that were forwarded in
+// and out of our node.
+func forwardNote(amtIn, amtOut uint64) string {
+	return fmt.Sprintf("incoming: %v msat outgoing: %v msat", amtIn, amtOut)
+}
+
+// forwardingEntry produces a forwarding entry with a zero amount which reflects
+// shifting of funds in our channels, and fees entry which reflects the fees we
+// earned form the forward.
+func forwardingEntry(forward *lnrpc.ForwardingEvent,
+	convert msatToFiat) ([]*HarmonyEntry, error) {
+
+	txid := forwardTxid(
+		forward.ChanIdIn, forward.ChanIdOut, forward.Timestamp,
+	)
+	note := forwardNote(forward.AmtInMsat, forward.AmtOutMsat)
+
+	fwdEntry, err := newHarmonyEntry(
+		int64(forward.Timestamp), 0, EntryTypeForward, txid, "", note,
```

Comment:
q: do we need the note? (as chan in/out is in the tx id)

Reply:
I think it's nice to have the amounts as well? Just so you can eyeball the amounts that you're forwarding.